packages/api/src/services/emp-state.ts

```diff
@@ -143,16 +143,20 @@ export const EmpState = (config: Config, dependencies: Dependencies) => {
   }
 
   async function updateAll(addresses: string[], startBlock?: number, endBlock?: number) {
-    await Promise.mapSeries(addresses, async (address: string) => {
-      const end = profile(`Update Emp state for ${address}`);
-      try {
-        return await updateOne(address, startBlock, endBlock);
-      } catch (err) {
-        console.error(err);
-      } finally {
-        end();
-      }
-    });
+    await Promise.map(
+      addresses,
+      async (address: string) => {
+        const end = profile(`Update Emp state for ${address}`);
+        try {
+          return await updateOne(address, startBlock, endBlock);
+        } catch (err) {
+          console.error(err);
+        } finally {
+          end();
+        }
+      },
+      { concurrency: 10 }
```

Comment:
can you make this a default value in the config for this service.  No need to expose it to env yet but one day we might want to